contrib/clojure-package/examples/infer/objectdetector/src/infer/draw.clj

```diff
@@ -0,0 +1,48 @@
+;; Licensed to the Apache Software Foundation (ASF) under one or more
+;; contributor license agreements.  See the NOTICE file distributed with
+;; this work for additional information regarding copyright ownership.
+;; The ASF licenses this file to You under the Apache License, Version 2.0
+;; (the "License"); you may not use this file except in compliance with
+;; the License.  You may obtain a copy of the License at
+;;
+;;    http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+
+(ns infer.draw
+	(:require 
+		[opencv4.colors.rgb :as rgb]
+		[opencv4.core :refer [FONT_HERSHEY_PLAIN imread imwrite new-point put-text! rectangle]]))
+
+(defn black-boxes! [img results]
+    (doseq [
```

Comment:
I feel like the formatting is a bit off in this function - I know if varies from editor to editor.  There is a tool to keep the style consistent between contributors, (I should run this more often I know :) )